components/Footer.vue

```diff
@@ -1,6 +1,58 @@
 <template>
-  <footer>
-    <!-- TODO: Create footer -->
-    <div class="py-40" />
+  <footer class="bg-surface-inverse-secondary">
+    <div class="mx-auto max-w-7xl px-6 pb-8 pt-8 lg:px-8 lg:pt-16">
+      <div class="mt-8 border-t border-outline-opaque pt-8 md:flex md:items-center md:justify-between">
+        <div class="flex space-x-6 md:order-2">
+          <a
+            v-for="item in socials"
+            :key="item.name"
+            :href="item.link"
+            target="_blank"
+            class="text-content-inverse-tertiary hover:text-gray-500"
+          >
+            <span class="sr-only">{{ item.name }}</span>
+            <component
+              :is="item.logo"
+              class="h-6 w-6"
+              aria-hidden="true"
+            />
+          </a>
+        </div>
+        <p class="mt-8 text-xs leading-5 text-content-inverse-tertiary md:order-1 md:mt-0">
+          &copy; 2023 Centrapay - All rights reserved.
+        </p>
+      </div>
+    </div>
   </footer>
 </template>
+
+<script setup>
+
+import facebookLogo from './icons/FacebookLogo.vue';
+import instagramLogo from './icons/InstagramLogo.vue';
+import twitterLogo from './icons/TwitterLogo.vue';
+import linkedInLogo from './icons/LinkedInLogo.vue';
```

Comment:
Instead of individually importing each logo I was wondering about globally importing our icons in `nuxt.config.js`:
```
components: {
    'dirs': [
      {
        'path': '~/components/icons',
        'global': true
      },
      '~/components'
    ]
  },
  ```
  
  Then you can just declare the logo as `facebook-logo` etc.